Packs/Code42/Integrations/Code42/integration-Code42.yml

```diff
@@ -320,7 +320,7 @@ script:
       description: The username to remove from the Departing Employee List.
```

Comment:
We shouldn't commit this I guess, but I don't really know how or where to .gitignore it.

Reply:
Actually - from looking at their .gitignore, it looks like they are ignoring another version of this file. They might not mind us to just edit the .gitignore at the root in this case, considering it is open source.